tests/core/utils/test_utils.py

```diff
@@ -34,3 +35,36 @@ def test_hostname():
             os.environ["RENKU_DOMAIN"] = renku_domain
         else:
             del os.environ["RENKU_DOMAIN"]
+
+
+def test_shorten_message():
+    """Test message is shorten correctly."""
+
+    short_message = "Lorem ipsum dolor sit amet"
+    long_message = (
+        "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt "
+        "ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco "
+        "laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in "
+        "voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat "
+        "non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."
+    )
+    max_line = 150
+
+    # The message is modified only when it's longer than the max line length
+    assert shorten_message(short_message, max_line) == short_message
+    assert shorten_message(long_message, max_line) != long_message
+    assert "\n" in shorten_message(long_message, max_line)
+    assert shorten_message(long_message, 0) == long_message
+    assert shorten_message(long_message, len(long_message)) == long_message
+    assert len(shorten_message(long_message, max_line)) > len(long_message)
+
+    # The message is trimmed when it's longer than the max body length
+    assert len(shorten_message(long_message, 0, body_length=max_line)) < len(long_message)
+    assert len(shorten_message(short_message, 0, body_length=max_line)) == len(short_message)
+    assert len(shorten_message(long_message, 0, body_length=0)) == len(long_message)
+    assert len(shorten_message(long_message, 0, body_length=len(long_message))) == len(long_message)
+
+    # The the message is wrapped correctly
+    assert len(shorten_message(long_message, max_line, body_length=max_line * 2)) < len(long_message)
+    assert len(shorten_message(short_message, max_line, body_length=max_line * 2)) == len(short_message)
+    assert shorten_message(long_message, max_line, body_length=max_line * 2).count("\n") == 1
```

Comment:
Should we add a test where `shorten_message(msg, ...).replace('\n', ' ')[:body_length - 3] == msg[:body_length - 3]` (or something similar) that makes sure the content of the message is preserved when wrapping it to multiple lines?